icons/clion/clion-plain-wordmark.svg

```diff
@@ -1,4 +1,3 @@
59h.008c0 2.46-1.614 4.543-4.329 4.543-2.609 0-4.382-2.145-4.382-4.602v-.062c0-2.457 1.618-4.535 4.324-4.535zm-81.512 8.8h8.16v1.36h-8.16v-1.36zm27.26 12.305v8.207h.865v-1.11a2.63 2.63 0 0 0 2.229 1.235c1.402 0 2.771-1.125 2.771-3.027v-.02c0-1.925-1.381-3.015-2.771-3.015a2.615 2.615 0 0 0-2.229 1.296v-3.566h-.865zm52.926.172v.953h.988v-.953h-.988zm-32.504.156v5.281c0 1.246-.65 1.891-1.54 1.891-.832 0-1.329-.398-1.814-1.125l-.635.55h-.002a2.696 2.696 0 0 0 2.434 1.384 2.43 2.43 0 0 0 1.729-.649 2.845 2.845 0 0 0 .724-2.07v-5.262h-.896zm13.726 0-.002 7.86h3.5c1.72 0 2.858-.797 2.858-2.145v-.023c0-1.082-.76-1.618-1.69-1.899a1.89 1.89 0 0 0 1.246-1.816v-.024a1.74 1.74 0 0 0-.502-1.257 2.876 2.876 0 0 0-2.06-.696h-3.35zm-4.13.313v1.754h-.807v.765h.806v3.508h.002c0 1.18.705 1.637 1.64 1.625a2.175 2.175 0 0 0 1.048-.246v-.727a1.818 1.818 0 0 1-.846.2c-.586 0-.974-.258-.974-.965v-3.395h1.84v-.765h-1.84v-1.754h-.87zm5.007.484h2.397c1.125 0 1.742.508 1.742 1.281l.004.024c0 .922-.748 1.402-1.815 1.402h-2.328V76.04zm-9.324 1.145c-1.609 0-2.787 1.351-2.787 3.023v.027a2.918 2.918 0 0 0 .81 2.137 2.905 2.905 0 0 0 2.112.883 2.985 2.985 0 0 0 2.388-1.086l-.556-.484a2.395 2.395 0 0 1-1.828.82 2.025 2.025 0 0 1-2.047-1.969h4.62a3.297 3.297 0 0 0 0-.281c0-1.695-.993-3.07-2.712-3.07zm32.871 0a2.206 2.206 0 0 0-1.99 1.14V77.31h-.863v5.82h.863V79.8a1.706 1.706 0 0 1 .475-1.285 1.714 1.714 0 0 1 1.269-.528c1.023 0 1.606.668 1.606 1.758v3.39h.869v-3.624c0-1.383-.823-2.328-2.229-2.328zm5.742.02c-1.148 0-1.978.663-1.978 1.663v.012c0 1.067 1.01 1.379 1.92 1.637.766.214 1.45.441 1.45 1.023v.027c0 .56-.503.918-1.202.918a3.366 3.366 0 0 1-2.014-.722l-.435.617a3.964 3.964 0 0 0 2.406.855c1.156 0 2.068-.652 2.068-1.738v-.023c0-1.047-.967-1.394-1.865-1.664-.77-.235-1.496-.45-1.496-1v-.024c0-.48.436-.844 1.123-.844a3.296 3.296 0 0 1 1.72.563l.39-.66a3.993 3.993 0 0 0-2.087-.64zm-18.933.003a2.456 2.456 0 0 0-1.44.434c-.426.288-.75.702-.93 1.187v-1.52h-.869v5.813h.87V80.81c0-1.758 1.028-2.668 2.298-2.668h.07v-.934zm3.32.016a4.84 4.84 0 0 0-2.111.484l.26.727a3.997 3.997 0 0 1 1.761-.426c1.094 0 1.737.535 1.737 1.586v.183a6.037 6.037 0 0 0-1.791-.25c-1.472 0-2.485.653-2.485 1.88v.027c0 1.194 1.09 1.812 2.168 1.804a2.557 2.557 0 0 0 2.108-1v.875h.828v-3.55a2.283 2.283 0 0 0-.594-1.7 2.567 2.567 0 0 0-1.881-.64zm-42.254.078 2.645 5.8c-.348.82-.676 1.09-1.196 1.09a1.853 1.853 0 0 1-.91-.214l-.293.683a2.583 2.583 0 0 0 1.239.293c.874 0 1.437-.438 1.937-1.64l2.436-6.004h-.918l-1.873 4.84-2.11-4.848h-.957zm46.535.008v5.8h.866v-5.8h-.866zm-27.3.609c1.156 0 1.761.918 1.851 1.98h-3.74c.113-1.14.878-1.98 1.889-1.98zm-22.75.043c1.101 0 2.046.875 2.046 2.246v.027c0 1.403-.92 2.239-2.047 2.239a2.18 2.18 0 0 1-2.105-2.254v-.02a2.175 2.175 0 0 1 2.105-2.238zm32.093 1.574h2.487v.012c1.375 0 2.089.496 2.078 1.336v.027c0 .875-.747 1.403-1.934 1.403h-2.63v-2.778zm13.615.64a6.014 6.014 0 0 1 1.708.247v.562c0 .926-.868 1.576-1.958 1.579-.774 0-1.455-.434-1.455-1.157v-.027c0-.731.615-1.203 1.705-1.203z"/>
```

Comment:
Shouldn't this also have the default color as fill?

Reply:
Maybe we should have original-wordmark with the default color and plain-wordmark with green color? But then it would be against TOS (I think) since the square in the middle has to be black. Not sure if that applies to us though

I want some more opinions on this though. @Panquesito7 @BenSouchet @kilian-paquier any thoughts on this?

Reply:
Hi,
In my opinion the current icons added by @kilian-paquier are good, shapes and colors, my only comment: I would add the `original-wordmark` version with the filled white area on the CL and the oblique line (in the square).

Just like the logo on the clion product homepage:
<img width="369" alt="Screenshot 2022-08-03 at 12 38 25" src="https://user-images.githubusercontent.com/17025808/182588659-e4323335-27c6-4f74-8bba-568c880d808d.png">

Reply:
Hi @BenSouchet  
Thanks for chiming in!

Okey, but what about this from the [Jetbrains Brand Assets](https://www.jetbrains.com/company/brand/#logos-and-icons-product-logos) page though? 
>The Black Box used in our logos must remain untouched – in terms of both its color and proportions to the text. Make sure that the Black Box always remains black and is never changed (e.g inverted) to another color, even from the official color palette of JetBrains.

Shouldn't we follow it when we can, or at the very least not encourage usage against their guildelines/terms?


This is what I think:
I don't think it's a good idea to fill in the background, nor to use the logo sent above. Mainly because it limits users by choosing the background color for them. Leaving it transparent gives more options. But also because transparent background is one of the requirements when adding icons to devicons as per the [SVG Standards](https://github.com/devicons/devicon/wiki/SVG-Standards). Furthermore I think we should use the "Product Logo" jetbrains provides in their brand guidelines to encourage correct usage of the icons/logos.

Reply:
@Snailedlt 
I disagree.
`original` version should always be the exact original logo of the tool / product, for jetbrains logos the white square is filled with white:
<img width="412" alt="Screenshot 2022-08-03 at 15 12 42" src="https://user-images.githubusercontent.com/17025808/182616779-fea6ed96-82ec-4a1d-ae29-7a2b5d81c417.png">
<img width="412" alt="Screenshot 2022-08-03 at 15 12 48" src="https://user-images.githubusercontent.com/17025808/182616792-38e0eb6a-f34e-488f-981e-e671ca85eb6e.png">

Taken from https://www.jetbrains.com/company/brand/

And the screen I share in my previous comment is from the product homepage of clion.

In the quote you shared `Black Box used in our logos must remain untouched` this means with the white part (for the `original` version)

Now for `plain` version these white part must be transparent otherwise in the devicon font generated it will be a full fill square (since for the font colors are ignored)

Reply:
And last point with `original` users **cannot** set a new color, `only for `plain` versions

Reply:
@BenSouchet Ohh, I see I misunderstood parts of your comment earlier.

To clarify:
When you said ` I would add the original-wordmark version with the filled white area on the CL and the oblique line (in the square).` and showed a picture of the logo, I misread and thought you meant the blue (not the white) background color should be added. Now that I know you meant the white parts inside the black box, I 100% agree with you. Didn't even notice it was transparent, so good catch on your part. 
I thought your main point was to add the blue colored background, or more specifically, that excact logo, which I disagreed with.

So if I understand you correctly now, you want the following change:
1. Add a new `original-wordmark` with the black box untouched (white spots must go back in)
So you mean like this then, right?
![](https://resources.jetbrains.com/storage/products/company/brand/logos/CLion.svg?_gl=1*xlhkho*_ga*MTI3NzMxNzc2NC4xNjQ5MDY0Mjc4*_ga_9J976DJZ68*MTY1OTUzMzM2Mi4zLjEuMTY1OTUzMzgzMy4w&_ga=2.39291491.1166350473.1659530529-1277317764.1649064278)

I agree with these changes ✔️ 

That said, isn't it still weird that the two plain icons have different colors? Shouldn't both have either black (like jetbrains wants), or green (to be in line with the main color of the logo)?

Reply:
> So if I understand you correctly now, you want the following change:
> Add a new original-wordmark with the black box untouched (white spots must go back in)
> So you mean like this then, right?

Yeah exactly 😊

> That said, isn't it still weird that the two plain icons have different colors? Shouldn't both have either black (like jetbrains wants), or green (to be in line with the main color of the logo)?

Since users can edit / change the color of the `plain` versions, it's not that important (in my opinion). I would say to continue to do as it has already been done for the others JetBrains tools / softwares.

I think I saw that @kilian-paquier  chose a unique color for every softwares, and apply this color on the `plain` version, and set black for the `plain-wordmark` version.
It doesn't bother me, on the wordmark version there is more black area than colored area (due to the wording).

Reply:
Okey, I've no objections to doing it this way, just thought it looked a bit weird at first glance.

Will wait for @Panquesito to comment here before continuing though :)

Reply:
So, what do we do ? 😄

Reply:
I say keep it as it is, unless @Panquesito7 is stronly against that :)

Reply:
@kilian-paquier @Snailedlt 
I still think that we should add an `clion-original-wordmark.svg` version with the filled white areas on the square. Like the one you shared in your previous comment @Snailedlt .

Reply:
Agreed!
@kilian-paquier Could you add a original-wordmark logo like explained above?

Reply:
@amacado agreed that this is a good solution, so I think we can go ahead and merge this once `clion-original-wordmark` has been added as specified above.

@kilian-paquier ^